Homework_1/Ex_7.py

```diff
@@ -0,0 +1,16 @@
+# Задача 7. Дано натуральное число. Требуется определить, является ли год с данным номером високосным.
+#           Если год является високосным, выведите YES, иначе выведите NO.
+#           Напомним, что в соответствии с григорианским календарем, год является високосным, если его номер кратен 4,
+#           но не кратен 100, а также, если он кратен 400
+
+year_num = int(input('Введите натуральное число: '))
+
+
+def is_leap_year(year):
+    if (year % 4 == 0 and year % 100 != 0) or year % 400 == 0:
```

Comment:
Скобки не нужны. Это условие и так будет выполняться первым.
```suggestion
    if year % 4 == 0 and year % 100 != 0 or year % 400 == 0:
```